File: bronze/10250/c++/main.cpp
#include <iostream>

int main()
{
    int h, w, n;
    int repeat;

    std::cin >> repeat;
    for (int i = 0; i < repeat; i++) {
        std::cin >> h >> w >> n;
        int height = (n % h == 0) ? h : (n % h);
        int width = (n % h == 0) ? (n / h) : (n / h +1);
        if (width >= 10) {
            std::cout << height << width << '\n';
        } else {
            std::cout << height << 0 << width << '\n';
        }
    }
    return 0;
}

File: bronze/10250/python/main.py

if (__name__ == "__main__"):
    repeat = int(input())

    for _ in range(repeat):
        h, w, n = map(int, input().split())
        height = h if (n % h == 0) else int(n % h)
        width = int(n / h) if (n % h == 0) else int(n / h +1)
        
        if (width >= 10):
            answer = str(height) + str(width)
        else:
            answer = str(height) + '0' + str(width)
        print(answer)